bundle/src/main/java/com/adobe/acs/tools/csv_asset_importer/impl/CsvAssetImporterServlet.java

```diff
@@ -280,24 +278,33 @@ private void updateProperties(final Map<String, Column> columns,
 
             final Column column = entry.getValue();
             final String valueStr = row[column.getIndex()];
-            final ModifiableValueMap properties = this.getMetadataProperties(asset,
-                    column.getRelPropertyPath());
+            final Node metaProps = this.getMetadataProperties(asset, column.getRelPropertyPath());
+            final String propName = column.getPropertyName();
 
             if (StringUtils.isNotBlank(valueStr)) {
+                if (metaProps.hasProperty(propName)) {
+                    Property prop = metaProps.getProperty(propName);
+                    if ((column.isMulti() && !prop.isMultiple()) || (!column.isMulti() && prop.isMultiple())) {
```

Comment:
Can add this after; technically a different issue from this GH issue.

Thanks!